projects/samples/howto/url/controllers/url/Makefile

```diff
@@ -0,0 +1,25 @@
+# Copyright 1996-2021 Cyberbotics Ltd.
```

Comment:
This file should be included to `projects/samples/howto/Makefile` and the description is missing in `samples-howto.md`.